File: beaker_kernel/subkernels/julia/procedures/reflectors/module.jl
{# target_type: __module__ #}
{# function_name: _beaker_reflect_module #}
function _beaker_reflect_module(_name::String, _value::Module)
    _full = string(nameof(_value))
    _tag = _name == _full ? "module(`$(_full)`)" : "module(`$(_full)` as `$(_name)`)"
    return Dict("tag" => _tag)
end

File: src/beaker_notebook/subkernels/python/procedures/reflectors/module.py
{# target_type: __module__ #}
{# function_name: _beaker_reflect_module #}
def _beaker_reflect_module(_name, _value):
    _full = getattr(_value, "__name__", None) or _name
    if _name == _full:
        _tag = f"module(`{_full}`)"
    else:
        _tag = f"module(`{_full}` as `{_name}`)"
    return {"tag": _tag}
